components/News/NewsGrid.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FunctionComponent } from "react";
+import NewsItem from "./NewsItem";
+import { TypeNews } from "../../generated/contentful";
+import Style from "./styles.module.scss";
+import clsx from "clsx";
+
+export interface NewsGridProps {
+  news: TypeNews[];
+  title?: string;
+  className?: string;
+}
+
+const NewsGrid: FunctionComponent<NewsGridProps> = ({
+  news,
+  title = null,
+  className = "",
+}) => {
+  return (
+    <>
+      {!!title && (
+        <div className={Style.newsTitle}>
+          <h3>{title}</h3>
+        </div>
+      )}
+      <div className={clsx(Style.newsLayout, className)}>
```

Comment:
It feels a bit like `className` should affect some wrapping grid around both title and item layout doesn't it? Maybe a better prop name is appropriate: `itemClassName`.